docs/config/lua/config/font_rules.md

```diff
@@ -45,48 +48,56 @@ return {
 }
 ```
 
-Here's an example from my configuration file;
+Here's an example from a configuration file;
 
 ```lua
 local wezterm = require 'wezterm'
 
 -- A helper function for my fallback fonts
 function font_with_fallback(name, params)
-  local names = { name, 'Noto Color Emoji', 'JetBrains Mono' }
+  local names = { name, 'Noto Color Emoji', 'Symbols Nerd Font Mono' }
   return wezterm.font_with_fallback(names, params)
 end
 
 return {
-  font = font_with_fallback 'Operator Mono SSm Lig Medium',
+  font = font_with_fallback('FiraCode', {weight = 'Light'}),
+  font_size = 8.0,
   font_rules = {
     -- Select a fancy italic font for italic text
     {
       italic = true,
-      font = font_with_fallback 'Operator Mono SSm Lig Medium Italic',
+      intensity = "Normal",
+      font = font_with_fallback('VictorMono', {
+        weight = 'Regular', italic = true }),
```

Comment:
I'd recommend installing the pre-requisites for building the docs, so that this can get automatically formatted.

```
cargo install mdbook mdbook-linkcheck mdbook-mermaid gelatyx
```

then run:

```
ci/build-docs.sh
```

it should auto-format the lua in these examples.